entity-types/infra-awsmqqueue/golden_metrics.yml

```diff
@@ -0,0 +1,126 @@
+consumerCount:
```

Comment:
There are 14 golden metric definitions. We are discouraged to define more than 10. Could you reduce this list a bit, or at least review them?

Reply:
it's all of the metrics we're ingesting for this entity, sorted by frequency. 

We have no idea about how useful these metrics are, but at least we'll try to make it so the dashboards are always populated.